package.json

```diff
@@ -4,6 +4,9 @@
   "description": "",
   "main": "lib/index.js",
   "scripts": {
+    "build": "babel lib -d lib --ignore 'lib/*.test.js'",
```

Comment:
Just to confirm, this is babeling everything (excluding tests) in `lib` and outputting it back to `lib`. Is this the intention? The pattern that I've seen more often is to have the non-transpiled code live in `src` and the transpiled output live in `lib`.

Reply:
ah, let's follow the convention @fritzmonkey points out.